kedro/framework/cli/utils.py

```diff
@@ -66,11 +66,7 @@ def python_call(
 def find_stylesheets() -> Iterable[str]:  # pragma: no cover
     """Fetch all stylesheets used in the official Kedro documentation"""
     css_path = Path(__file__).resolve().parents[1] / "html" / "_static" / "css"
-    return (
-        str(css_path / "copybutton.css"),
-        str(css_path / "qb1-sphinx-rtd.css"),
-        str(css_path / "theme-overrides.css"),
-    )
+    return (str(css_path / "copybutton.css"),)
```

Comment:
Just found out that this function exists after doing this PR, and I noticed that it's used in all the starters...